django_project/core/settings/contrib.py

```diff
@@ -144,6 +144,15 @@
         'extra_context': {
             'media': 'screen, projection',
         },
+    },
```

Comment:
These changes should be in project not contrib. Contrib is for contributed code (written by third parties), project is for code written by us.